utils/site_settings.py

```diff
@@ -41,6 +43,10 @@ def is_site_closed(request):
   return False
 
 
+def review_requested():
+  return literal_eval(site_settings_get("REVIEW_REQUESTED", default=False, use_cache=True))
```

Comment:
Why `literal_eval`?

Reply:
site_settings_get returns a string, so eval is needed to convert it to bool; however, eval is somewhat unsafe, so literal_eval is used.